src/app/core/components/group-nav/group-nav.component.html

```diff
@@ -5,12 +5,12 @@
     </span>
     <alg-switch mode="white"></alg-switch>
   </div> -->
-  <!-- <div class="group-self" (click)="onSelectYourself($event, 0)">
+  <div class="group-self" (click)="onSelectYourself()">
     <span class="anonym-group">
       <img src="assets/images/Male.svg" style="width: 3.3333rem; height: 3.3333rem;" />
     </span>
-    <span class="group-subtitle" [ngClass]="{'selected': selectedGroup == 0 ? true : false}">Yourself</span>
-  </div> -->
+    <span class="group-subtitle" [ngClass]="{'selected': selectedGroup == 0}">Yourself</span>
+  </div>
```

Comment:
The click and the link selection should be handle 100% in html which now if you current page matches the link.

Reply:
Done